docs/storage/clone_api.md

```diff
@@ -1,7 +1,10 @@
 # Clone API
 
-The `clone.kubevirt.io` API Group defines resources for cloning KubeVirt objects. Currently, the only supported cloning
-type is `VirtualMachine`, but more types are planned to be supported in the future (see future roadmap below).
+The `clone.kubevirt.io` API Group defines resources for cloning KubeVirt objects.
+
+This API is useful both for cloning Virtual Machines,
+but also for creating "golden snapshots" that would be used as a template for creating new VMs
```

Comment:
In my opinion not sure 'but' is the best word here..
'but' is more fitting if you use 'not only for....' 'but also'
here you put both for ...., but also
seems a bit odd.

Reply:
Also I think you should distinguish between the possible sources of the clone and the use cases of it, and probably put those in different paragraphs :)